### PR TITLE
Ft checkout page

### PR DESCRIPTION
**Pull request by wamani-will**
1.Desktop version is ok
2. input area stores data in local storage
3. confirm button on click should return an alert message "Your order is being processed, thank you"

### CODE REVIEW

File: UI/script.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ function showAlert() {
+   alert("Your order is being processed, thank you");
+   
+ };
+
+//Declaring input information
+var name = document.getElementById("name").value;
+var location = document.getElementById("location").value;
+var phone = document.getElementById("phone-no").value;
+var email = document.getElementById("email").value;
```
Comment (jennizalwango) [inline UI/script.js]: @wamani-will  you can use `let` instead of `var`

File: UI/script.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+// storage of input information
+localStorage.setItem("Name",name);
+localStorage.setItem("Location",location);
+localStorage.setItem("Phone",phone);
+localStorage.setItem("Email",email);
```
Comment (jennizalwango) [inline UI/script.js]: @wamani-will  these 4 lines of code can also be written like
``` 
let personDetails  = {
"Name": name,
"Location": location,
"Phone": phone,
"Email": email
}
localStorage.setItem("pdetails", JSON.stringfy(productDetails))
```

The reason why l use `JSON.stringfy` is to convert the object into a JSON object.

**Review by jennizalwango — COMMENTED** [review]
Thanks for trying

File: UI/script.js (hunk 1 above)
Comment (wamani-will) [reply UI/script.js]: Okay

File: UI/script.js (hunk 2 above)
Comment (wamani-will) [reply UI/script.js]: Thanks, I saw the JSON idea but I thought it was out of vanilla JavaScript

**Comment by wamani-will** [comment]
Thanks but I see I haven't unleashed my potential, I hope to improve next time🙏 thanks for facilitating us🙏

**Pull request merged**